run_linux.go

```diff
@@ -359,26 +359,36 @@ func runSetupBuiltinVolumes(mountLabel, mountPoint, containerDir string, builtin
 			}
 			initializeVolume = true
 		}
-		stat, err := os.Stat(srcPath)
-		if err != nil {
-			if !os.IsNotExist(err) {
-				return nil, err
+		// Check if srcPath is a symlink
+		stat, statErr := os.Lstat(srcPath)
```

Comment:
nit: no reason not to just leave this as err.

Reply:
the smoke test is unhappy when I use err, it claims one of the stat `err` is unused, which is not the case. Going to leave it as `statErr`.